ospd_openvas/wrapper.py

```diff
@@ -1264,7 +1285,11 @@ def exec_scan(self, scan_id, target):
             value='An OpenVAS Scanner was started for %s.' % target,
         )
 
-        cmd = ['sudo', 'openvas', '--scan-start', openvas_scan_id]
+        if self.sudo_available:
+            cmd = ['sudo', 'openvas', '--scan-start', openvas_scan_id]
+        else:
+            cmd = ['openvas', '--scan-start', openvas_scan_id]
+
```

Comment:
Can be simplified as above.

Maybe it would even better to extract calling openvas into an own method/function e.g.
```python3
def call_openvas(args, use_sudo=False, niceness=None):
   cmd = args

   if use_sudo:
       cmd = ['sudo'] + cmd

   if niceness is not None:
      cmd = ['nice', ...] + cmd

    try:
        subproces. ....
    ...
```
but i am not 100 percent sure about that because we are calling different functions of subrocess at the moment with slightly different arguments.